jsonschema2md/__init__.py

```diff
@@ -316,9 +325,20 @@ def _construct_description_line(self, obj: dict[str, Any], add_type: bool = Fals
                 description_line.append(_("Cannot contain unevaluated properties."))
 
         if "$ref" in obj:
+            url = urlsplit(obj["$ref"])
+            if url.fragment and not url.path:
+                ref_link = f"#{quote(url.fragment[1:])}"
+            elif self.base and (url.netloc == self.base or url.path.startswith(self.base)):
+                ref_file = url.path.removeprefix(self.base).removesuffix(".json")
```

Comment:
This should probably be delegated to a function that will parse difference "naming conventions" (.schema.json, .json, .json.schema (if that's a thing), etc.)

Reply:
+1 :-)